src/napari_cryoet_data_portal/_reader.py

```diff
@@ -211,10 +266,74 @@ def read_annotation(annotation: Annotation, *, tomogram: Optional[Tomogram] = No
     else:
         attributes["name"] = f"{tomogram.name}-{name}"
     attributes["metadata"] = annotation.to_dict()
-    attributes["face_color"] = OBJECT_COLOR.get(name.lower(), DEFAULT_OBJECT_COLOR)
+    attributes["face_color"] = OBJECT_COLOR.get(name, DEFAULT_OBJECT_COLOR)
```

Comment:
Ideally, I want the object-name to color map to be stable across runs. So that's why I went for a single fallback color over cycling through the colors based on the order the object names are encountered.

I also considered hashing the object name with the modulo to get some variety with stability. Any other thoughts?

Reply:
There seem to be way more object IDs (from the Gene Ontology Cellular Component identifier) than object names, so I guess another alternative is to to enumerate and map those, since I assume those will be much more stable?

Reply:
Resolved by using the `object_id` instead. There is still a fallback color, but hopefully we should not run into that in the wild.